docs/release/trg-0/trg-1-06.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.06 - Administrator's Guide (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+Administrator's Guide is intended for administrators that install and operate the solution.
+
+## Description
+
+- The Admin Guide **has to** be stored in the folder: ```docs/admin/```
+- Admin Guide is available with the software at the same time.
+- Admin Guide is correct and up to date; English is a must.
+- The documentation is of appropriate maturity to be handed over to any Operations Company with global business practice intentions.
+- In Markdown format. Link to the document available. Must point to the leading repository within Tractus-X.
+
+## Best Practice
```

Comment:
Again, this should guide on how to ensure high quality and not how to participate in reviews.

Things that could be aligned via best practice: 

- Type of diagrams used for specific aspects
- Specific topics that must be covered

Reply:
@vialkoje can we propose general guidance for the aspects?:
 - Type of diagrams used for specific aspects
 - Specific topics that must be covered

Reply:
delete lines 28 to end.
new text for best practices (bulletpoints):
The Admin guide is in sync with the latest released version of the component. It is checked and updated, once an up-revision is foreseen.
The content is of appropriate maturity, so that any Operations Company with global business practice intentions is invited to use and build on it.
Anchor a link to your Admin guide within your readme.md (see TRG 1.01; https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1)